hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/RouterWebServices.java

```diff
@@ -206,50 +181,33 @@ protected Map<String, RequestInterceptorChainWrapper> getPipelines() {
    */
   @VisibleForTesting
   protected RESTRequestInterceptor createRequestInterceptorChain() {
+    RESTRequestInterceptor pipeline = null;
```

Comment:
This code is very similar to the other one.
As we are refactoring, can we clean this up?

Reply:
Thanks for your help reviewing the code, I will continue to refactor this part of the code.